Tools/CISupport/ews-build/steps_unittest.py

```diff
@@ -9249,6 +9249,69 @@ def test_success(self):
         return self.runStep()
 
 
+class TestFindModifiedSaferCPPExpectations(BuildStepMixinAdditions, unittest.TestCase):
+    def setUp(self):
+        return self.setUpBuildStep()
+
+    def tearDown(self):
+        return self.tearDownBuildStep()
+
+    def configureStep(self):
+        self.setupStep(FindModifiedSaferCPPExpectations())
+
+    def test_success(self):
+        self.configureStep()
+        self.setProperty('builddir', 'wkdir')
+        self.setProperty('buildnumber', 1234)
+
+        commit_diff = '''
+diff --git a/Source/WebCore/SaferCPPExpectations/NoUncountedMemberCheckerExpectations b/Source/WebCore/SaferCPPExpectations/NoUncountedMemberCheckerExpectations
```

Comment:
Can we test a diff where a line is added and removed in the same file? Ideally one where the diff looks like a line was replaced. Something where we had a line that did this:
`inspector/agents/worker/WorkerAuditAgent.h -> inspector/agents/worker/WorkerWorkerAgent.h`

Reply:
Yep, I will add a test for the following diff. Is that what you're looking for?
```
diff --git a/Source/WebCore/SaferCPPExpectations/NoUncountedMemberCheckerExpectations b/Source/WebCore/SaferCPPExpectations/NoUncountedMemberCheckerExpectations
index 8a2d2375b8d2..f7ebc3b11b94 100644
--- a/Source/WebCore/SaferCPPExpectations/NoUncountedMemberCheckerExpectations
+++ b/Source/WebCore/SaferCPPExpectations/NoUncountedMemberCheckerExpectations
@@ -51,7 +51,7 @@ inspector/InspectorShaderProgram.h
 inspector/InspectorStyleSheet.cpp
 inspector/InspectorStyleSheet.h
 inspector/InspectorWebAgentBase.h
-inspector/agents/worker/WorkerAuditAgent.h
+inspector/agents/worker/WorkerWorkerAgent.h
```